platform-services.md

```diff
@@ -51,17 +51,17 @@ non-graphical, data-only information. Examples include:
 *   Network information
 *   Sensors
 
-You can also use the [`HostMessages`](https://docs.flutter.io/flutter/services/HostMessages-class.html)
+You can also use the [`PlatformMessages`](https://docs.flutter.io/flutter/services/PlatformMessages-class.html)
 class to invoke APIs from third-party SDKs that run on iOS and Android devices.
 
 To communicate between the
-[`HostMessages`](https://docs.flutter.io/flutter/services/HostMessages-class.html) class and
+[`PlatformMessages`](https://docs.flutter.io/flutter/services/PlatformMessages-class.html) class and
 platform-specific APIs in the host, you must add the following Flutter components into
 the host's view hierarchy.
 
-+ **On iOS:** Add a [`FlutterViewController`](https://github.com/flutter/engine/blob/master/sky/shell/platform/ios/framework/Headers/FlutterViewController.h)
++ **On iOS:** Add a [`FlutterViewController`](https://github.com/flutter/engine/blob/master/shell/platform/darwin/ios/framework/Headers/FlutterViewController.h)
```

Comment:
Not sure, others will know. Even if it's the latter, I'm saying that link breakage is one important signal that the website needs to change. Often it's not just the link that needs updating...this PR is a good example. It's a cluster of changes, including subtracting Mojo, that were triggered by link breakage. Without the breakage, the flutter and website repos can drift apart silently and smoothly, with no red flags. Normally, I agree that avoiding errors and hurtles is totally excellent. In this case, there would need to be some other reliable indicator or sync mechanism.